ulauncher/api/__init__.py

```diff
@@ -1,8 +1,12 @@
 # flake8: noqa
 # pylint: disable=unused-import
 from ulauncher.api.extension import Extension
-from ulauncher.api.result.result import Result
-from ulauncher.api.result.searchable_result import SearchableResult
-from ulauncher.api.result.small_result import SmallResult
-from ulauncher.api.result.extension_result import ExtensionResult
-from ulauncher.api.result.extension_small_result import ExtensionSmallResult
+from ulauncher.api.result import Result
+
+
+class ExtensionResult(Result):
```

Comment:
This is only temporary because I ported extensions to v6 using ExtensionResult and ExtensionSmallResult. Before release these should be removed.